tests/onnx/benchmarking/dataset_definitions.yml

```diff
@@ -1495,3 +1495,17 @@ datasets:
       annotation_file: object_detection/streams_1/high/annotations/instances_glb2bcls3.json
     annotation: mscoco_detection_high_3cls.pickle
     dataset_meta: mscoco_detection_high_3cls.json
+
+  - name: imagenette2-320
```

Comment:
Using any files in OpenVINO backend from ONNX backend is not good practice. Could you move `dataset_definition.yml` to a share part of the code, for example in the `tests/shared/data`?

Reply:
Created specific `ov_dataset_definition.yml` for  OpenVINO backend